develop/plone/serving/traversing.rst

```diff
@@ -4,7 +4,7 @@
 
 .. admonition:: Description
 
-    Plone content is organized to a tree. Traversing means looking up
+    Plone content is organized on a tree. Traversing means looking up
     content from this tree by path. When HTTP request hits a Plone
```

Comment:
I am not a native English speaker, so I am not sure, for me both meaning 'to' and 'on' are sounding weird, at least in the context here, should it be 'as' or 'in' like 'Plone content is organized as/in a tree' ?

Like I said I'm not sure, too :)